Base: Add Latin Extended-A to font Katica Regular 12

krkk opened the pull request:
0100-017E: https://www.unicode.org/charts/PDF/U0100.pdf

![Font Editor window showing new glyphs!](https://user-images.githubusercontent.com/16520278/155409179-1a33c6ac-b55e-4cfa-ad87-f016a7d96d67.png)
![Preview window showing text: “Zażółć gęślą jaźń // Pchnąć w łódź jeża lub ośm skrzyń fig”](https://user-images.githubusercontent.com/16520278/155403796-40b84ffc-3463-4e50-b3ff-b76a0b197522.png)

pinging @xexxa as you’re probably interested in this :)

Xexxa commented:
> pinging @Xexxa as you’re probably interested in this :)

You are correct in assuming that! :-)

First of all, awesome! Then the nitpicking:

I guess you based it somewhat on Katica Regular 10? If that is the case that's obviously good, but some glyphs like 0126, 0132 is probably from the day when that font was only 7px wide and should probably be adjusted in Katica Regular 10 and not propagated to other sizes.

In the chart and the reference font I just checked 0126 has a base form as a normal H + the extra space for the line, so it should probably be 1 px wider.

In 0132 the "I" and "J" is not joined in the charts nor the reference fonts I just checked.

I would recommend a quick "does this character look weird because we only had 7 px to work with"-check (and fix it upstream in Katicia Regular 10 if you feel like it).

krkk commented:
Changes since last push:

- made U+1026 and U+1032 wider in Katika 12
  ![preview](https://user-images.githubusercontent.com/16520278/155610739-58de61f4-3796-4ce5-a247-1b5b1b903272.png)

- made U+1026, U+1032, U+1052 and U+1053 wider in Katika 10
  ![preview](https://user-images.githubusercontent.com/16520278/155610577-8b6f65ea-bbcf-40f5-87c9-8f031945b31d.png)

Xexxa reviewed (approved):
Better Unicode coverage AND improvements in current characters, could not be much better! :partying_face:
